contracts/VeJoeStaking.sol

```diff
@@ -0,0 +1,267 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
```

Comment:
Can we use a more recent version of solidity for new contracts? This is possible with hardhat see https://hardhat.org/guides/compile-contracts.html#multiple-solidity-versions

Reply:
done! see my comment in https://github.com/traderjoe-xyz/joe-core/pull/28#discussion_r799957850